mcs/class/System.IO.Compression/Test/System.IO.Compression/ZipTest.cs

```diff
@@ -239,6 +239,31 @@ public void ZipCreateArchive()
 		}
 
 		[Test]
+		public void ZipEnumerateEntriesModifiedTime()
```

Comment:
This test passes before the patch too, so it isn't really testing whether the fix works.